articles/libraries/_includes/_verifying_migration.md

```diff
@@ -1,9 +1,12 @@
 ### Verifying your migration
 
-Once you have migrated your codebase, if you would like to be sure that your applications are no longer calling the legacy endpoints, you can go to the [Dashboard](${manage_url}/#/tenant/advanced) under **Tenant Settings > Advanced** then scroll down to **Migrations** and toggle off the Legacy Lock API switch. Turning off this switch will disable the deprecated Lock / Auth0.js endpoints for your tenant, preventing them from being used at all.
+Once you have migrated your codebase, you should no longer see [deprecation notes in your logs](/errors/deprecation-errors).
```

Comment:
It's better to be consistent and refer to this either as "deprecation notes" or "deprecation errors"